lua/lualine/components/tabs/tab.lua

```diff
@@ -14,6 +14,10 @@ end
 --- of the tab.
 ---@return string
 function Tab:label()
+  local custom_tabname = vim.fn.gettabvar(self.tabnr, 'tabname')
```

Comment:
You should use `vim.t.tabname`.

Reply:
I think I should use `vim.fn.gettabvar(self.tabnr, 'tabname')`. Otherwise, all tabs would have the  same name.

Reply:
My bad it should be
```lua
vim.t[self.tabnr].tabname
```